src/pages/calendar/components/CalendarWeek.jsx

```diff
@@ -0,0 +1,182 @@
+import styled, { css } from 'styled-components'
+import { useBreakpointValue, getBreakpointsStyles } from '../../../styles/breakpoints'
+import { useAppThemeContext } from '../../../styles/theme/provider'
+
+export const CalendarWeek = () => {
+  const { setThemeType } = useAppThemeContext()
+  setThemeType('dark')
+
+  // const mon = useBreakpointValue({
+  //   desktopValue: 'mon',
+  //   tabletValue: 'mon',
+  //   mobileValue: 'm',
+  // })
+  // const tue = useBreakpointValue({
+  //   desktopValue: 'tue',
+  //   tabletValue: 'tue',
+  //   mobileValue: 't',
+  // })
+  // const wed = useBreakpointValue({
+  //   desktopValue: 'wed',
+  //   tabletValue: 'wed',
+  //   mobileValue: 'w',
+  // })
+  // const thu = useBreakpointValue({
+  //   desktopValue: 'thu',
+  //   tabletValue: 'thu',
+  //   mobileValue: 't',
+  // })
+  // const fri = useBreakpointValue({
+  //   desktopValue: 'fri',
+  //   tabletValue: 'fri',
+  //   mobileValue: 'f',
+  // })
+  // const sut = useBreakpointValue({
+  //   desktopValue: 'sut',
+  //   tabletValue: 'sut',
+  //   mobileValue: 's',
+  // })
+  // const sun = useBreakpointValue({
+  //   desktopValue: 'sun',
+  //   tabletValue: 'sun',
+  //   mobileValue: 's',
+  // })
+
+  const days = [
+    {
+      id: '1',
+      value: useBreakpointValue({ desktopValue: 'mon', tabletValue: 'mon', mobileValue: 'm' }),
```

Comment:
1. No need to use hooks in this way.
First that you commented it better. You can ask chat GPT why?
2. Use each of text from translations library.
3. Create seaprate hook for it -> useWeekDays